selenium/test/exchanges/management.js

```diff
@@ -44,7 +44,7 @@ describe('Exchange management', function () {
       ["/", "amq.fanout", "fanout"],
       ["/", "amq.headers", "headers"],
       ["/", "amq.match", "headers"],
-      ["/", "amq.rabbitmq.event", "topic"],
+//      ["/", "amq.rabbitmq.event", "topic"],
```

Comment:
FTR, this exchange is declared and used by a plugin that is no longer enabled as of this PR.